keps/sig-architecture/4330-compatibility-versions/README.md

```diff
@@ -507,6 +508,27 @@ compatibility version can be set to.
 
 Alpha APIs may not be enabled in conjunction with compatibility version.
 
+### Discovery
+
+Discovery will [enable](https://github.com/kubernetes/kubernetes/blob/7080b51ee92f67623757534f3462d8ae862ef6fe/staging/src/k8s.io/apiserver/pkg/util/openapi/enablement.go#L32) the group versions matching the compatibility version.
```

Comment:
Something possibly tangential that affects my interpretation of this statement: is APIDiscovery intended to return a different response depending on which apiserver you talk to? In the context of other features like StorageVersionProxy would this change?

Reply:
This is being discussed right now on the UVIP KEP: https://github.com/kubernetes/enhancements/pull/4441#pullrequestreview-1844787411.  I think UVIP is a good forum for this one.